javascript/react-verbs/src/verbs/Convert/Convert.tsx

```diff
@@ -31,105 +31,100 @@ import { getColumnType } from '../shared/TypingFunction/TypingFunction.js'
 /**
  * Provides inputs for a Convert step.
  */
-export const Convert: React.FC<StepComponentProps> = memo(function Convert({
-	step,
-	store,
-	table,
-	onChange,
-	input,
-}) {
-	const internal = useMemo(() => step as ConvertStep, [step])
-	const tbl = useLoadTable(
-		input || internal.input[NodeInput.Source]?.node,
-		table,
-		store,
-	)
-	const [inputColumnDate, setInputColumnDate] = useState<boolean>()
+export const Convert: React.FC<StepComponentProps<ConvertArgs>> = memo(
+	function Convert({ step, store, table, onChange, input }) {
+		const tbl = useLoadTable(
+			input || step.input[NodeInput.Source]?.node,
+			table,
+			store,
+		)
+		const [inputColumnDate, setInputColumnDate] = useState<boolean>()
 
-	const handleTypeChange = useHandleDropdownChange(
-		internal,
-		'args.type',
-		onChange,
-	)
+		const handleTypeChange = useHandleDropdownChange(
+			step,
+			'args.type',
+			onChange,
+		)
 
-	const handleRadixChange = useHandleTextfieldChange(
-		internal,
-		'args.radix',
-		onChange,
-	)
+		const handleRadixChange = useHandleTextfieldChange(
+			step,
+			'args.radix',
+			onChange,
+		)
 
-	const handleComboBoxChange = useCallback(
-		(
-			_e: any,
-			option: IComboBoxOption | undefined,
-			_index: number | undefined,
-			value: string | undefined,
-		) => {
-			const update = cloneDeep(step)
-			set(update, 'args.formatPattern', option ? option.key : value)
-			onChange && onChange(update)
-		},
-		[step, onChange],
-	)
+		const handleComboBoxChange = useCallback(
+			(
+				_e: any,
+				option: IComboBoxOption | undefined,
+				_index: number | undefined,
+				value: string | undefined,
+			) => {
```

Comment:
fluent ComboBoxChangeHandler?

Reply:
Possibly, it's only used once, and the usage isn't super generic. If we moved to a callback-based update pattern I could see this

Reply:
isn't this the standard onChange definition for fluent ComboBox? we define handler types for the rest of these (i.e., previous fluent typings comment)

Reply:
Yes, but the update mattern may be different (will it use index ever? is the value fallback always consistent?)

Reply:
Why is this different [than this](https://github.com/microsoft/data-wrangling-components/blob/9e15859ff84b0522c9cf46607f4473cfdcf7e250/javascript/react-verbs/src/common/hooks.ts#L23)?

The callback interface needs to be compliant with the fluent callback either way - you can name the variables whatever you want.